src/voter/mod.rs

```diff
@@ -302,13 +304,13 @@ pub struct RoundData<Id, Timer, Input, Output> {
 	pub outgoing: Output,
 }
 
-struct Buffered<S: Sink> {
+struct Buffered<S, I> {
```

Comment:
Note: the signature of the `Sink` trait has changed. The item is now a parameter of the trait instead of an associated type.

Reply:
Nice - it didn't exist when I wrote this code.